ding/model/template/q_learning.py

```diff
@@ -663,6 +663,9 @@ def reshape(d):
                 d = d.reshape(T, B, *d.shape[1:])
             return d
 
+        # NOTE(rjy): the initial input shape will be (T,B,N),
+        #            means encoder or head should process T*B trajectories,
+        #            so here we need such fn to reshape for encoder or head
```

Comment:
but T and B dimension can be both treated as `batch_size` in encoder and head, i.e., independent and parallel processing, se we reshape...